main/WebServer.cpp

```diff
@@ -15147,7 +15147,7 @@ namespace http {
 					if (!result.empty())
 					{
 						std::map<int, int> _directions;
-						int wdirtabletemp[17][8];
+						int wdirtabletemp[17][8] = {};
```

Comment:
I would have expected a conversion to std::array from you here *g* ;)

Reply:
:).

The reason for the other std::array conversions is because bool[d] = {} does not work. std::array<bool, d> = {} does.

Reply:
I tested it by just declaring it as

`						std::array<std::array<int, 8>, 17> wdirtabletemp = {};`

and removing the for loop to zero the array

Or you could use the fill method like:

`wdirtabletemp .fill({});`

Reply:
that's quite verbose. = {} does the same thing. If using fill might as well set to 0...

Reply:
Yes, that's why the first declaration is perfect too. I was experimenting here trying to fill it whole array with a number other then zero... failed without creating a for loop... But not needed in this case ;)